libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.html

```diff
@@ -32,47 +32,53 @@
 
 <ng-template #resultControlsTemplateRef>
   <div
-    *ngIf="searchText && (!showActionsArea || searchResults.length > 0)"
     class="sky-autocomplete-results"
+    [class.sky-screen-reader-only]="
```

Comment:
I might be misunderstanding the issue, but instead of refactoring, could we check if the autocomplete results are "open" like we're doing for the `aria-owns` property? https://github.com/blackbaud/skyux/blob/200b77425edc88388f5525428d59eeb0289baf0e/libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.html#L8

Reply:
Removed the autocomplete changes.